### ISSUE TITLE
Add TexMacs to the recipe

### ISSUE THREAD

**Issue by icaoberg** [icaoberg/singularity-latex#2]
Add [TexMacs](http://www.texmacs.org/tmweb/home/welcome.en.html) to the recipe.

**Comment by icaoberg** [comment]
@syakante did you?

**Comment by syakante** [comment]
@icaoberg no, i added and then commented out the code that should do it because it wouldn't work and I can't figure out why :( gonna get on it today (had exam this week so couldn't earlier sorry)

**Comment by icaoberg** [comment]
![animation-movie-minions-uBuzWfwVcadRC](https://media2.giphy.com/media/uBuzWfwVcadRC/giphy.gif?cid=e1bb72ff5ca96fb45166314132616aeb)
